fedimint-dbtool/src/dump.rs

```diff
@@ -221,6 +223,9 @@ impl DatabaseDump {
         }
 
         if let Some(cfg) = self.client_cfg.clone() {
+            self.serialized
```

Comment:
Now we serialize module configs separately from the client config and also have the binary encoding of the module configs in the complete config. Instead we could use `JsonClientConfig` here.